djangocms_admin_style/sass/components/_content.scss

```diff
@@ -45,6 +45,45 @@ body {
     clear: both;
 }
 
+//application groups
+#applications-group {
+    h2 {
+        margin-bottom: 30px;
+    }
+    .module {
+        overflow: visible;
+    }
+    table {
+        margin-bottom: 30px;
+        border-radius: 0;
+        box-shadow: $base-box-shadow;
+        thead {
+            th {
+                font-size: 14px;
+                font-weight: normal;
+                height: 27px;
+                padding: 5px 10px 3px !important;
+            }
+        }
+        td,
+        th {
+            border-left: 0;
+            background: none;
+        }
+        td {
+            font-size: 16px;
```

Comment:
you probably don't need `font-size` and `line height` here, since it's specified below. Unless there is something else in `td` (apart from `p`-s and `a`-s)